Collections/01_CTC API Testing Local.postman_collection.json

```diff
@@ -430,9 +430,15 @@
 											"",
 											"var moment = require('moment')",
 											"",
+											"// Use a Base36 number so that it's alphanumeric and less",
+											"// likely to trigger credit card disclosure scanners",
+											"var intConRef = _.random(0, 6140942214464815497215)",
```

Comment:
This max number is `zzzzzzzzzzzzzz` so it's the maximum number that can fit into 14 digits in base 36